docs/pipelines.md

```diff
@@ -429,7 +429,7 @@ There are a lot of scenarios where `WhenExpressions` can be really useful. Some
 
 ### Guard `Task` execution using `Conditions`
 
-**Note:** `Conditions` are deprecated, use [`WhenExpressions`](#guard-task-execution-using-whenexpressions) instead.
+**Note:** `Conditions` got deprecated in pipelines v0.16.0, use [`WhenExpressions`](#guard-task-execution-using-whenexpressions) instead.
```

Comment:
Since documents are versioned, I feel like the current wording is ok, but we could include a link to the deprecation table, e.g.

`Conditions` are \[deprecated\](./deprecations.md)

Reply:
Ahh didn't even know that existed.
That will make my life much easier in the future.
I will update the PR after lunch